source/Gamerules.h

```diff
@@ -35,6 +39,11 @@ class Gamerules {
 
 	// Load a gamerules node.
 	void Load(const DataNode &node);
+	void Save(DataWriter &out) const;
```

Comment:
This one is also unused, but we'll need it eventually anyway for rules that are customized in-game. I'll kick it out of this PR if we want, though.